packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# 52.17.0 - October 5, 2021
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Custom templates for vscode ([PR #3563](https://github.com/forcedotcom/salesforcedx-vscode/pull/3563))
+
+## Fixed
+
+#### docs
+
+- New Topic BYO Template ([PR #3565](https://github.com/forcedotcom/salesforcedx-vscode/pull/3565))
```

Comment:
Something like "Documentation for custom templates in VS Code" would be better. I'll let others suggest something better :)